<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmAgregarEvento</class>
 <widget class="QWidget" name="FrmAgregarEvento">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>702</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agregar Evento</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>431</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Progrmacion de Condiciones</string>
   </property>
   <widget class="QRadioButton" name="RdbAnd">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y (AND)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RdbOr">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>O (OR)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CboListaEntradas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TxtValorCondicion">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="SpnValorCondicion">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
   </widget>
   <widget class="QComboBox" name="CboValorCondicion">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Condiciones">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Condiciones</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnAgregarCondicion">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="LstCondiciones">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>81</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="RdbNone">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Programacion De Acciones</string>
   </property>
   <widget class="QComboBox" name="CboListaSalidas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Acciones</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CboValor">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnAgregarAccion">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="LstAcciones">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>411</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnFormatLCD">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Formatear LCD</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="BtnGuardar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>670</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar y Cerrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCerrar">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>670</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TxtNombreEvento">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre Del Evento</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>BtnGuardar</zorder>
  <zorder>BtnCerrar</zorder>
  <zorder>TxtNombreEvento</zorder>
  <zorder>label_3</zorder>
  <zorder>SpnValorCondicion</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
